.github/workflows/bashlib.sh

```diff
@@ -90,9 +90,11 @@ build-instrumented-assets() {
 setup-postgres() {
   say "::group::Initialize database"
   psql "postgresql://superset:superset@127.0.0.1:15432/superset" <<-EOF
-    DROP SCHEMA IF EXISTS sqllab_test_db;
+    DROP SCHEMA IF EXISTS superset CASCADE;
+    DROP SCHEMA IF EXISTS sqllab_test_db CASCADE;
+    DROP SCHEMA IF EXISTS admin_database CASCADE;
```

Comment:
This change shouldn't impact the CI, but they are useful when testing things locally.